### PR TITLE
Create books collection.

### PR DESCRIPTION
**Pull request by Lanr3waju**
# Books Collections with Functions to Add and Remove Books
### This PR does the following;

- Removes non-dynamic elements from the `js` file.
- Implement DOM manipulation in various parts of the project to load the books dynamically
- JS `Objects`, `Arrays`, & `Object Arrays` to store books.
- Implement the use of Arrow Functions `=>` and `{destructure}` methods to match the modern conventions.
- Made use of various `JS` objects and methods such as; `node.createElement()`, `node.AppendChild()`, `node.replaceChild()`, etc. to avoid the use of `innerHTML` for security considerations.
- Implemented `array.forEach()`, through the course of the project to iterate through `objects` in the `Objects Array`; thus making it possible to manipulate the objects and produce the desired result.
- Made use of `if Statements` to match specific use cases.
- Implement the use of `eventListeners` to make 'add' & 'remove' buttons functional.
- Implement the use of `array. find()` & `parseInt` to locate the specific books to remove when the appropriate button is clicked.
- The `JSON` API `stringify` & `parse` were implemented, to enable JS Objects to be used on windows local storage.
- The `node.localStorage` `set()` & `get()` methods were used to store and retrieve updated books.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,6 +1,129 @@
+const body = document.querySelector('body');
+const booksContainer = document.querySelector('.books-container');
+const bookSection = document.querySelector('.main-section');
+const inputBookTitle = document.querySelector('.book-title');
+const inputBookAuthor = document.querySelector('.book-author');
+const newBookForm = document.querySelector('.new-book-form');
+const addBookButton = document.querySelector('.add-book');
```
Comment (adejam) [inline js/index.js]: It's best to use `id` when targetting one element alone. This gives more clarity to the code.

File: js/index.js [hunk 2]
```diff
@@ -1,6 +1,129 @@
...
+  if (localStorage.length > 0) {
+    books = updatedBooks;
+  }
+};
+
+const handleEventListeners = () => {
+  addBookButton.addEventListener('click', pushBooks);
+  newBookForm.addEventListener('submit', event => {
+    event.preventDefault();
+  });
```
Comment (adejam) [inline js/index.js:118]: Nothing seems to happen when the add book feature gets called. I guess the functionality to add a new book is being triggered elsewhere, this looks hacky and can be improved on

**Review by adejam — CHANGES_REQUESTED** [review]
## Looks good overall. Let's fix the issues mentioned in the review

Also Let's make our code more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and [Solid](https://www.digitalocean.com/community/conceptual_articles/s-o-l-i-d-the-first-five-principles-of-object-oriented-design)

File: js/index.js (hunk 1 above)
Comment (Lanr3waju) [reply js/index.js]: refactored, thanks

**Pull request merged**